app/graphql/types/query_type.rb

```diff
@@ -100,5 +114,17 @@ def venue(id:)
     def venueByName(name:)
       Venue.where('name ILIKE ?', "%#{name}%")
     end
+
+    def subEvents
+      SubEvent.all
+    end
+
+    def subEvent(id:)
+      SubEvent.find(id)
+    end
+
+    def subEventByName(name:)
+      SubEvent.where('name ILIKE ?', "%#{name}%")
+    end
```

Comment:
Good addition here. We can chat with the FE team about other sub_event related stuffs we might need. This all looks good to me tho!